### PR TITLE
better command line argument handling

### PR DESCRIPTION
**Pull request by msgmaxim**
- also use `~/.loki/storage` as the default database location
- use that location for ssl keys/certificate

### CODE REVIEW

File: httpserver/main.cpp [hunk 1]
```diff
@@ -63,33 +64,57 @@ bool parseLogLevel(const std::string& input,
...
         uint16_t lokid_rpc_port = 22023;
 
         po::options_description desc;
-        desc.add_options()("lokid-key", po::value(&lokid_key_path),
-                           "")("db-location", po::value(&db_location),
-                               "")("output-log", po::value(&log_location), "")(
-            "log-level", po::value(&log_level_string),
-            "")("version,v", po::bool_switch(&print_version),
-                "")("lokid-rpc-port", po::value(&lokid_rpc_port));
+        // clang-format off
```
Comment (BeaudanBrown) [inline httpserver/main.cpp]: Nice I didn't know you could do that

File: httpserver/main.cpp [hunk 2]
```diff
@@ -63,33 +64,57 @@ bool parseLogLevel(const std::string& input,
...
+        desc.add_options()
+            ("lokid-key", po::value(&lokid_key_path), "Path to the Service Node key file")
+            ("db-location", po::value(&db_location),"Path to the Storage Server database location")
+            ("output-log", po::value(&log_location), "Path to the log file")
```
Comment (BeaudanBrown) [inline httpserver/main.cpp]: We should also store the logfile in the base path as well further down?
```
if (vm.count("output-log")) {
.....
  log_location += ".out";
.....
  auto sink = logging::add_file_log(log_location); // put base path/log_location?
```

**Comment by BeaudanBrown** [comment]
Closes #164

File: httpserver/main.cpp (hunk 2 above)
Comment (msgmaxim) [reply httpserver/main.cpp]: OK, but we might do that in a separate PR.

**Pull request merged**